api/agent/agent.go

```diff
@@ -652,6 +668,20 @@ func (c *container) WriteStat(ctx context.Context, stat drivers.Stat) {
 	for key, value := range stat.Metrics {
 		span.LogFields(log.Uint64("fn_"+key, value))
 	}
+	// Fn polls for stats as long as container lives,
+	// so we have to write stats only once
+	if c.callID != "" {
+		err := c.ds.UpdateCallMetrics(ctx,
+			c.appName, c.callID,
+			stat.Metrics["cpu_user"],
+			uint64(stat.Metrics["mem_max_usage"]/(1024*1024)),
```

Comment:
would like to store all stats from `cherryPick`, aside from the brittleness of doing this. imo we should store json blob in db or something.

Reply:
this isn't correctly computing max. e.g. if Java does GC and then stats are collected, this value is lower than the previous -- we should store the histogram and later do processing, it's way better than restricting at the base.